.github/pull_request_template.md

```
@@ -0,0 +1,26 @@
+<!--
+Thank you for contributing to PyZn!
+So that your PR can be handled effectively, please populate the following fields
+-->
+
+**Category**: 
+> One of: Bugfix / Feature / Code style update / Refactoring Only / Build related changes /  Documentation / Other (please specify)
+
+**Overview**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)